src/relayclient/RelayClient.ts

```diff
@@ -300,6 +300,7 @@ export class RelayClient {
         nonce: senderNonce,
         tokenAmount: tokenAmount,
         tokenGas: tokenGas,
+        enableQos: transactionDetails.enableQos ?? false,
```

Comment:
```suggestion
        enableQos: !!transactionDetails.enableQos,
```
Is the same

Reply:
ok, but why would you pick one over the other? (curious, for me  `??` is more readable than `!!`)

Reply:
Actually because `transactionDetails.enableQos ?? false` is the same as `transactionDetails.enableQos ? true : false` and that is redundant since any value on javascript different from `undefined, null, '', false` is true so you can check that easily by using `!!transactionDetails.enableQos` but again is not a blocker and the 3 options are ok.